### ISSUE TITLE
Update or remove litegraph.d.ts

### ISSUE THREAD

**Issue by space-nuko** [jagenjo/litegraph.js#357]
Please either update or remove `litegraph.d.ts`, the latest version has many new fields that give typescript errors, and its much harder to override this package's provided types than to version private types for this library

**Comment by moritz89** [comment]
The best long-term solution would be to provide an MR that fixes any broken types :smile:

**Comment by space-nuko** [comment]
Okay I can do that, I have a project where I added some fields that were missing, but I might not have got them all

**Comment by space-nuko** [comment]
I might get reactions of "that's excessive" but I ended up just porting litegraph to TypeScript instead. I found a lot of typing-related errors and undocumented fields/callbacks. Plus nodes can be created in a typed way now.

Don't know how long I'll maintain it for but the code is here (I'm only developing it for now so I can use it in a different project) https://github.com/space-nuko/litegraph.ts
